app-tracking-protection/vpn-impl/src/main/res/values/strings-vpn.xml

```diff
@@ -196,12 +186,12 @@
     <string name="atp_ActivityPastWeek">Past 7 Days</string>
     <string name="atp_ActivityTitle">App Tracking Protection</string>
     <string name="atp_MostRecentActivityTitle">All Recent Activity</string>
-    <string name="atp_ActivityRecentActivity">Recent Activity</string>
+    <string name="atp_ActivityRecentActivity">Activity</string>
```

Comment:
We've modified the content of this string but left the ID intact. But we haven't removed the translated strings.
The string is also in string-vpn-untranslated.xml.
Translations are not going to be correct in the end